SixthLesson/DecoratorStream/DecoratorStream/Decorators/PasswordRequestBeforeReading.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.IO;
+using DecoratorStream.Model;
+using DecoratorStream.View;
+
+namespace DecoratorStream.Decorators
+{
+    public class PasswordRequestBeforeReading : BaseDecoratorStream
```

Comment:
Need to rename. At least, add **Decorator** at the end of the name. Or rename to e.g. **RequestPasswordDecorator**

Reply:
Renamed RequestPasswordDecorator